tests/test_agent.py

```diff
@@ -340,7 +348,7 @@ def return_tuple(_: list[ModelMessage], info: AgentInfo) -> ModelResponse:
             ModelRequest(
                 parts=[
                     RetryPromptPart(
-                        content='Plain text responses are not permitted, please include your response in a tool call',
+                        content='Please include your response in a tool call.',
```

Comment:
```suggestion
                        content='Please call a tool.',
```
? more similar to below. Doesn't really matter